Formula/mesa.rb

```diff
@@ -110,6 +111,38 @@ def install
       --enable-sysfs
       --with-llvm-prefix=#{Formula["llvm@4"].opt_prefix}
     ]
+    
+    if build.with?("only-cpu")
+      args = %W[ 
+      CFLAGS=#{ENV.cflags}
+      CXXFLAGS=#{ENV.cflags}
+      --disable-silent-rules
+      --disable-dependency-tracking
+      --prefix=#{prefix}
+      --sysconfdir=#{etc}
+      --localstatedir=#{var}
+      --enable-opengl
+      --disable-gles1
+      --disable-gles2
+      --disable-va
+      --disable-xvmc
+      --disable-vdpau
+      --enable-shared-glapi
+      --disable-texture-float
+      --enable-gallium-llvm
+      --disable-llvm-shared-libs
+      --with-gallium-drivers=swrast,swr
+      --disable-dri
+      --with-dri-drivers=
+      --disable-egl
+      --with-egl-platforms=
+      --disable-gbm
+      --disable-glx
+      --disable-osmesa
+      --enable-gallium-osmesa
+      --with-llvm-prefix=#{Formula["llvm@4"].opt_prefix}
+      ]
+    end
```

Comment:
Instead of redefining `args` completely, determine common arguments and leave them in the original definition and then add
```
if `build.with? "gpu"`
...
else
...
end
```

Reply:
Ok. I did.